src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -16,9 +17,9 @@
   (let [{:keys [workspace-id on-back]} props
         workspaces-list (:workspaces-list @state)]
     [:div {}
-     (when (:importing? @state)
-       [comps/Blocker {:banner "Importing..."}])
-
+     (when (:blocking? @state)
+       [comps/Blocker {:banner (if (:blocking-text @state)
+                                 (:blocking-text @state) "Please wait...")}])
```

Comment:
A pattern I wrote for this case (needing multiple blockers) a while back was to just key the blocker on the text itself.  I don't know if that code is still in the codebase, but it would look like this:

    (when (:blocking-text @state)
      [comps/Blocker {:banner (:blocking-text @state)}])

Then when you want to start blocking, you `assoc :blocking-text "desired message"`, and when you want to stop blocking, you `dissoc :blocking-text` (or `assoc` it to `nil`).